src/Native/LIBMFNative/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project (LIBMFNative)
```

Comment:
Should we drop the `LIB` prefix?  On Unix machines the binary is going to be named `libLIBMFNative`. Is that what we want?

Reply:
I prefer to have libLIBMFNative. LIBMF is the name of the external package (something like tensorflow).

Reply:
Sounds good to me. I wasn't sure if that was part of the official name, or just a prefix.